ghost/content.md

```diff
@@ -40,23 +46,6 @@ $ docker run -d --name some-ghost -p 3001:2368 -v /path/to/ghost/blog:/var/lib/g
 $ docker run -d --name some-ghost -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost %%IMAGE%%:0.11-alpine
 ```
 
-### Breaking changes
```

Comment:
I think maybe we should keep this section.  It makes the `/var/lib/ghost/content` vs `/var/lib/ghost` clear.

Reply:
I thought the section was redundant with `## STATEFUL` below it also explicitly specifying the proper path, so to make it clear I re-arranged and worded it:
> . . . mount your data to the appropriate path in the predecessor container (see below)

> ## Stateful
> 
> Mount your existing content. In this example we also use the Alpine base image.
> 
> ### Ghost 1.x.x
> 
> ```console
> $ docker run -d --name some-ghost -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost/content ghost:1-alpine
> ```
> 
> ### Ghost 0.11.xx
> 
> ```console
> $ docker run -d --name some-ghost -p 3001:2368 -v /path/to/ghost/blog:/var/lib/ghost ghost:0.11-alpine
> ```

And then the next section likewise reiterates the proper paths again

> ### Docker Volume
> 
> Alternatively you can use a [data container](http://docs.docker.com/engine/tutorials/dockervolumes/) that has a volume that points to `/var/lib/ghost/content` (or /var/lib/ghost for 0.11.x) and then reference it: